marker_namespace: "litter_markers"
marker_scale:
  x: 0.2
  y: 0.2
  z: 0.2
marker_color:
  r: 1.0
  g: 0.0
  b: 0.0
  a: 1.0
